pkg/gui/options_menu_panel.go

```diff
@@ -59,9 +59,9 @@ func (gui *Gui) handleCreateOptionsMenu() error {
 	for i, binding := range bindings {
 		binding := binding // note to self, never close over loop variables
 		menuItems[i] = &menuItem{
-			displayStrings: []string{GetKeyDisplay(binding.Key), gui.displayDescription(binding)},
+			displayStrings: []string{GetKeyDisplay(binding.KeyMod), gui.displayDescription(binding)},
 			onPress: func() error {
-				if binding.Key == nil {
+				if binding.KeyMod.Key == nil {
```

Comment:
I wonder why a binding's key could be nil in the first place. Makes me wonder if KeyMod could be nil

Reply:
I don't think it can be null in any valid scenario. getKey could return nil but it is after a log.Fatal call.

Reply:
yeah, I'm happy to just remove this if statement and wait and see if anything happens